templates/cluster/eks-standalone-cp/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: eks-standalone-cp
```

Comment:
standalone/hosted would not be applicable for eks, as it's essentially hosted by aws.